subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -121,7 +120,7 @@
 
  */
 @CacheableTask
-public class Test extends AbstractTestTask implements JavaForkOptions, PatternFilterable, Reporting<TestTaskReports> {
```

Comment:
Can you explain why this actually broke?

Reply:
I have no clear idea of why this happens. All I could see is there are 2 `getReports()` methods that get compiled in and for some reasons one is on the task class where `implements Reporting<TestTaskReports>` is declared and the other one is on the `Test_Decorated` class.